src/components/SignUp/Account.vue

```diff
@@ -3,45 +3,57 @@
       <div class="account-field">
         <v-text-field
           class="account-inputs"
+          :rules="firstNameRules"
           placeholder="First Name"
           hint="ex: John"
           persistent-hint
+          required
           outlined
         /><span class="asterisk">*</span>
       </div>
       <div class="account-field">
         <v-text-field
           class="account-inputs"
+          :rules="lastNameRules"
           placeholder="Last Name"
           hint="ex: Smith"
           persistent-hint
+          required
           outlined
         /><span class="asterisk">*</span>
       </div>
       <div class="account-field">
         <v-text-field
           class="account-inputs"
+          :rules="emailRules"
           placeholder="Email Address"
           hint="ex: JohnSmith@gmail.com"
           persistent-hint
+          required
           outlined
         /><span class="asterisk">*</span>
       </div>
       <div class="account-field">
         <v-text-field
           class="account-inputs"
+          v-model="userPassword"
+          :rules="passwordRules"
           placeholder="Password"
           hint="ex: ABCabc123"
           persistent-hint
+          required
           outlined
         /><span class="asterisk">*</span>
       </div>
       <div class="account-field">
         <v-text-field
           class="account-inputs"
+          v-model="confirmUserPassword"
+          :rules="confirmPasswordRules"
           placeholder="Confirm Password"
           hint="ex: ABCabc123"
```

Comment:
the hint doesn't match up with the rules, missing symbols

Reply:
Those rules will not be used to validate that field. There needs to be some code that checks the 1st password typed in and see if it matches this one.